litmus-portal/frontend/src/components/PredifinedWorkflows/data.ts

```diff
@@ -20,13 +20,58 @@ export default [
     urlToIcon:
       'https://hub.litmuschaos.io/api/icon/1.7.0/generic/node-memory-hog.png',
     chaosWkfCRDLink:
-      'https://raw.githubusercontent.com/litmuschaos/chaos-charts/master/workflows/node-memory-hog/workflow.yaml',
+      'https://raw.githubusercontent.com/ishangupta-ds/chaos-workflows/master/Argo/argowf-native-node-memory-hog.yaml',
 
     gitLink:
-      'https://github.com/litmuschaos/chaos-charts/blob/master/workflows/node-memory-hog/workflow.yaml',
+      'https://github.com/ishangupta-ds/chaos-workflows/blob/master/Argo/argowf-native-node-memory-hog.yaml',
     provider: 'MayaData',
     description: 'Injects a memory spike on a node',
-    totalRuns: 5300,
+    totalRuns: 4300,
+    isCustom: false,
+  },
+  {
+    workflowID: '3',
+    title: 'pod-cpu-hog',
+    urlToIcon:
+      'https://hub.litmuschaos.io/api/icon/1.7.0/generic/pod-cpu-hog.png',
+    chaosWkfCRDLink:
+      'https://raw.githubusercontent.com/ishangupta-ds/chaos-workflows/master/Argo/argowf-native-pod-cpu-hog.yaml',
+
+    gitLink:
+      'https://github.com/ishangupta-ds/chaos-workflows/blob/master/Argo/argowf-native-pod-cpu-hog.yaml',
+    provider: 'MayaData',
+    description: 'Injects a CPU spike on a pod',
+    totalRuns: 5000,
+    isCustom: false,
+  },
+  {
+    workflowID: '4',
+    title: 'pod-memory-hog',
+    urlToIcon:
+      'https://hub.litmuschaos.io/api/icon/1.7.0/generic/pod-memory-hog.png',
+    chaosWkfCRDLink:
+      'https://raw.githubusercontent.com/ishangupta-ds/chaos-workflows/master/Argo/argowf-native-pod-memory-hog.yaml',
+
+    gitLink:
+      'https://github.com/ishangupta-ds/chaos-workflows/blob/master/Argo/argowf-native-pod-memory-hog.yaml',
+    provider: 'MayaData',
+    description: 'Injects a memory spike on a pod',
+    totalRuns: 3005,
+    isCustom: false,
+  },
+  {
+    workflowID: '5',
+    title: 'pod-delete',
+    urlToIcon:
+      'https://hub.litmuschaos.io/api/icon/1.7.0/generic/pod-delete.png',
+    chaosWkfCRDLink:
+      'https://raw.githubusercontent.com/ishangupta-ds/chaos-workflows/master/Argo/argowf-native-pod-delete.yaml',
+
+    gitLink:
+      'https://github.com/ishangupta-ds/chaos-workflows/blob/master/Argo/argowf-native-pod-delete.yaml',
+    provider: 'MayaData',
+    description: 'Deletes a pod',
+    totalRuns: 6700,
```

Comment:
Addition of extra workflow is not a part of this PR. It can go with another PR

Reply:
Let's not change any existing workflow manifest

Reply:
Yeah but these are ready so i have added them, just will update the username to litmuschaos in the next PR, and previously we were using the CronWorkflow template, but now we have changed to the Workflow template after discussion with @ksatchit , so we need to make these changes, because the UI has been updated to parse the old template, removing these changes will crash the frontend. @rajdas98

Reply:
Removed these changes. @rajdas98

Reply:
With these manifests schedule now is not possible also the workflow run mapping with workflow id. @rajdas98 @gdsoumya cc: @ksatchit